src/ast.jl

```diff
@@ -142,7 +142,7 @@ end
 function makeleaf(ctx, srcref, k::Kind, value; kws...)
     graph = syntax_graph(ctx)
     if k == K"Identifier" || k == K"core" || k == K"top" || k == K"Symbol" ||
-            k == K"globalref" || k == K"Placeholder"
+            k == K"globalref" || k == K"Placeholder" || k == K"MacroName"
```

Comment:
```suggestion
            k == K"globalref" || k == K"Placeholder" || k == K"MacroName" ||
            k == "StringMacroName" || k == K"CmdMacroName"
```

I assume this should be synchronized with `_value_string`